src/components/NFTBook/SignatureBanner/Mockup.vue

```diff
@@ -46,24 +60,85 @@ import LottieAnimation from 'lottie-web-vue';
 
 import SignatureAnimation from './signature-animation.json';
 
-const signaturePage = require('./signature-page.png');
 const deviceFrame = require('./device-frame.png');
 
+const NAMES = ['Alice', 'Bob', 'Carol', 'David', 'Edmond'];
+const MESSAGES = [
+  '不怕，我們有彼此。',
+  'Thank you for your support.',
+  '感謝支持！',
+];
+
 export default {
   name: 'NFTBookSignatureBanner',
   components: {
     LottieAnimation,
   },
   computed: {
-    signaturePageImage() {
-      return signaturePage;
-    },
     deviceFrameImage() {
       return deviceFrame;
     },
     animationData() {
       return SignatureAnimation;
     },
+    textClasses() {
+      return [
+        'absolute',
+        'left-[40px] desktop:left-[50px]',
+        'text-[#333]',
+        'font-serif',
+        'w-[160px] desktop:w-[200px]',
+      ];
+    },
+  },
+  mounted() {
+    this.animateText();
+  },
+  methods: {
+    animateText() {
+      const timeline = this.$gsap.gsap.timeline({ repeat: -1 });
+
+      NAMES.map(name => `${name},`).forEach((str, index) => {
+        const chars = str.split('');
+
+        // Add char by char
+        chars.forEach(char => {
+          timeline.to(
+            {},
+            {
+              duration: 0.1,
+              onComplete: () => {
+                this.$refs.nameText.textContent += char;
+              },
+            }
+          );
+        });
+
+        MESSAGES[index % MESSAGES.length].split('').forEach(char => {
```

Comment:
Would be nice if we randomized the order

Reply:
Updated